src/CommonLib/Processors/WebClientServiceProcessor.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Win32.SafeHandles;
+using SharpHoundCommonLib.OutputTypes;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SharpHoundCommonLib.Processors
+{
+    public class WebClientServiceProcessor(ILogger log = null)
+    {
+        private readonly ILogger _log = log ?? Logging.LogProvider.CreateLogger("WebClientServiceProcessor");
+
+        // Define constants
+        public const uint MAXIMUM_ALLOWED = 0x02000000;
```

Comment:
I believe these match the naming style of the library calls, but not sure we want to copy that naming style into our managed code